### PR TITLE
Feature/daef 179 show terms of use screen on first application start

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR introduces "Terms of use" screen which is shown to the user on first application start and user must accept terms in order to use the application.
As a part of this PR "Terms of use" screen was implemented along with the UI logic and acceptance test for this feature.

terms-of-use-not-accepted
terms-of-use-accepted

## TODO:
- [ ] Add real "Terms of use" text

### CODE REVIEW

**Comment by nikolaglumac** [comment]
@DominikGuzei will check it out. Thanks for reporting!

**Comment by nikolaglumac** [comment]
@DominikGuzei fixed :)
fix

**Comment by DominikGuzei** [comment]
@nikolaglumac if you have the rights you can merge this PR although the CI is not yet approving it (or trigger another build)

File: app/api/CardanoClientApi.js [hunk 1]
```diff
@@ -64,12 +64,33 @@ const setUserLocaleInLocalStorage = (locale) => new Promise((resolve, reject) =>
...
 });
 
-const unsetUserLocaleInLocalStorage = () => new Promise((resolve) => {
+const unsetUserLocaleFromLocalStorage = () => new Promise((resolve) => {
   localStorage.remove('userLocale', () => {
     resolve();
   });
 });
 
+const getTermsOfUseAcceptanceFromLocalStorage = () => new Promise((resolve, reject) => {
```
Comment (nikolaglumac) [inline app/api/CardanoClientApi.js]: Agreed! I will create a new card for it ;)

**Pull request merged**